Makefile

```diff
@@ -415,7 +415,7 @@ test: | $(JUNIT_REPORT)
 	mkdir -p $(dir $(JUNIT_UNIT_TEST_XML))
 	set -o pipefail; \
 	$(MAKE) --keep-going pilot-test mixer-test security-test broker-test galley-test common-test \
-	|& tee >($(JUNIT_REPORT) > $(JUNIT_UNIT_TEST_XML))
+	2>&1 tee >($(JUNIT_REPORT) > $(JUNIT_UNIT_TEST_XML))
```

Comment:
You still need the pipe tho
`2>&1 | tee >($(JUNIT_REPORT) > $(JUNIT_UNIT_TEST_XML))`